SU-HSDO/suhumsci
# HSD8-686 stage file proxy

# READY FOR REVIEW

# Summary
- Use stage_file_proxy module on dev, stage and local
- disable dblog
- custom module to record 404 & 403 reports with custom pages at the same url as the ones provided by dblog

# Needed By (Date)
- 7/31

# Urgency
- Medium

# Steps to Test
1. checkout this branch
1. `composer install --prefer-source`
1. `blt drupal:sync --site=[siteofyourchoice] --partial`
1. ensure theres a "Top 404 Pages" and "Top 403 Pages" reports pages under the "Reports" menu in the toolbar
1. try a random url and ensure it appears in the 404 page list.
1. try to view `/admin` when logged out in an incognito window and ensure it appears in the 403 list.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
